content/api/plugins/compiler.md

```diff
@@ -116,3 +116,16 @@ This a reference guide to all the event hooks exposed by the `Compiler`.
 ## Examples
 
 ?> Adds examples of usage for some of the above events
```

Comment:
You can replace this TODO with a short description of the example you're providing.